bpmn-model/src/main/java/io/zeebe/model/bpmn/builder/AbstractCallActivityBuilder.java

```diff
@@ -38,4 +39,11 @@ public B calledElement(String calledElement) {
     element.setCalledElement(calledElement);
     return myself;
   }
+
+  public B zeebeProcessId(String processId) {
```

Comment:
It is like I proposed in #3174. Instead of using the BPMN `calledElement` attribute, I decided to put this into the Zeebe extension element `calledElement` because
* we want to support more attributes like resolving the process id by an expression and as for now - all Zeebe specifics attributes are in extension elements, so I want to bundle all attributes into one element
* `calledElement` is not the original attribute from the BPMN spec - it's modified in the Zeebe/Camunda model API to support string instead of QName
* `calledElement` is not very clear that you should put the BPMN process id in